packages/core/admin/admin/src/pages/AuthPage/components/Register/index.js

```diff
@@ -108,214 +110,220 @@ const Register = ({ fieldsToDisable, noSignin, onSubmit, schema }) => {
           validationSchema={schema}
           validateOnChange={false}
         >
-          {({ values, errors, handleChange }) => (
-            <Form noValidate>
-              <Main>
-                <Column>
-                  <Logo />
-                  <Box paddingTop={6} paddingBottom={1}>
-                    <Typography as="h1" variant="alpha">
-                      {formatMessage({
-                        id: 'Auth.form.welcome.title',
-                        defaultMessage: 'Welcome!',
-                      })}
-                    </Typography>
-                  </Box>
-                  <CenteredBox paddingBottom={7}>
-                    <Typography variant="epsilon" textColor="neutral600">
-                      {formatMessage({
-                        id: 'Auth.form.register.subtitle',
-                        defaultMessage:
-                          'Your credentials are only used to authenticate yourself on the admin panel. All saved data will be stored in your own database.',
-                      })}
-                    </Typography>
-                  </CenteredBox>
-                </Column>
-                <Stack size={7}>
-                  <Grid gap={4}>
-                    <GridItem col={6}>
-                      <TextInput
-                        name="firstname"
-                        required
-                        value={values.firstname}
-                        error={
-                          errors.firstname
-                            ? formatMessage({
-                                id: errors.firstname,
-                                defaultMessage: 'This value is required.',
-                              })
-                            : undefined
-                        }
-                        onChange={handleChange}
-                        label={formatMessage({
-                          id: 'Auth.form.firstname.label',
-                          defaultMessage: 'Firstname',
+          {({ values, errors, handleChange, submitCount }) => {
+            if (submitCount > 1 && isEmpty(errors)) {
```

Comment:
I am not a huge fan of this do we have any other way to access the number of times the user tried to submit the form ?

Reply:
I guess you could remove the validationSchema prop and set the errors imperatively so you could set a state in order to have the number of times the form has been submitted.

Also I am not so sure about what is does here? Since we only send the event when the form does not have any error it means that you are trying to deal with the errors from the back-end right? Why don't you do this in the submit function instead?